pipeline/pipeline.go

```diff
@@ -36,21 +33,17 @@ type (
 )
 
 const (
-	// FunctionSetupEnvironment - identifier function configuration your structure
-	FunctionSetupEnvironment  = iota
-	FunctionSetupHocon        = iota
-	FunctionSetupJson         = iota
-	FunctionSetupYaml         = iota
-	FunctionSetupDefault      = iota
-	FunctionSetupVault        = iota
-	FunctionSetupConfigServer = iota
+	SmartConfiguring = true
+	/*DirtyConfiguring - flag*/
+	DirtyConfiguring = false
 
 	sourceFileInDisk   = 0
 	sourceFielInServer = 1
 	sourceFileNotUsed  = 2
 
 	notSupportedTypeError = "not supported type "
 
+	/*EmptyAdditionalPrefix - prefix for setup before all values can be empty*/
```

Comment:
prefix for setting up / prefix to setup
And not clear what you mean here, maybe you missed a comma after values or i didn't get it?

Reply:
i remove this comment